fetch-in-react/src/app.jsx

```diff
@@ -12,10 +12,10 @@ class App extends React.Component {
 
   componentDidMount() {
     /* your code here */
-    this.setState({ isLoading: false });
     fetch('https://jsonplaceholder.typicode.com/users')
       .then(res => res.json())
       .then(data => this.setState({ users: data }));
+    this.setState({ isLoading: false });
```

Comment:
Close. Try moving this state update next to your user data update

Reply:
🧠